src/ui/public/courier/saved_object/saved_object.js

```diff
@@ -55,11 +55,10 @@ export default function SavedObjectFactory(es, kbnIndex, Promise, Private, Notif
     let customInit = config.init || _.noop;
 
     // optional search source which this object configures
-    self.searchSource = config.searchSource && new SearchSource();
+    self.searchSource = config.searchSource ? new SearchSource() : null;
```

Comment:
Switched to undefined which has the added benefit of maintaining logic of the old code.